clients/middleware/client.go

```diff
@@ -0,0 +1,20 @@
+package middleware
+
+import (
+	"github.com/stitchfix/flotilla-os/state"
+	"net/http"
+)
+
+type Client interface {
```

Comment:
Is AclClient a better name to i) make this clearer and ii) in case there are other future middlewares?

Reply:
This is generic. We can call it ACL client in flotilla-sf.